public/index.html

```diff
@@ -3,35 +3,174 @@
-      The build step will place the bundled scripts into the <body> tag.
-
-      To begin the development, run `npm start` or `yarn start`.
-      To create a production bundle, use `npm run build` or `yarn build`.
-    -->
-  </body>
-</html>
+    <body class="app">
+    <header class="header flex__between"> 
+        <section class="logo flex__between">
+            <img src="./covering/logos_spotify-icon.svg" alt="Logo" width="65" height="65"/>
+            <span class="main__txt">Home</span>
+        </section>
+         
+        <section class="about__user">
+            <span class="txt about__user__first"><b>Polina Guseva</b></span>
+            <a href="#">
+                    <img class="icon__u" src="./covering/user__avatar.jpg" alt="User__avatar" width="60" height="60" />
+                </a>
+        </section>
+    </header>
+    <section class="head__menu__fixed">
+        <section class="head__menu flex__between">
+            <a href="#" class="tag txt">Podcasts</a>
+            <a href="#" class="tag txt">Playlists</a>
+            <a href="#" class="tag txt">Genres</a>
+            <a href="#" class="tag txt">Performers</a>
+        </section>
+    </section>
+    <aside class="menu flex__between">
+        <aside class="flex">
+            <a href="#" class="icon">
+                <img src="./covering/user.svg" alt="User" width="65" height="65"/>
+             </a>
+            <a href="#" class="icon">
+                <img src="./covering/search.svg" alt="Search" width="65" height="65" />
+            </a>
+            <a href="#" class="icon">
+                <img src="./covering/playlist.svg" alt="Playlists" width="65" height="65"/>
+            </a>
+            <a href="#" class="icon">
+                <img src="./covering/popular.svg" alt="Popular playlists" width="65" height="65"/>
+            </a>
+            <a href="#" class="icon">
+                <img src="./covering/like.svg" alt="Like" width="60" height="60"/>
+            </a>
+        </aside>
+        <aside class="flex">
+            <a href="#" class="icon">
+                <img src="./covering/settings.svg" alt="Settings" width="65" height="65" />
+            </a>
+            <a href="#" class="icon">
+                <img src="./covering/help.svg" alt="Help" width="65" height="65" />
+            </a>
+        </aside>   
+    </aside>
+    <aside class="albums">
+        <section class="banner">
+            <a class="big__banner">
+                <section class="big__banner__inf">
+                    <section>
+                        <span class="main__txt txt__black"><b>Ludovico Einaudi</b></span>
+                        <span class="txt__black txt__24">Underwater</span>
+                    </section>
+                    <span class="txt txt__black">Путешествие в прошлое</span>
+                    <button class="play__button__banner"> <img alt="Play" src="./banner/play__banner.svg" width="15" height="15"/>Слушать</button>
```

Comment:
По умолчанию у кнопки type=submit
Стоит явно указать нужный тип
```suggestion
                    <button class="play__button__banner" type="button"> <img alt="Play" src="./banner/play__banner.svg" width="15" height="15"/>Слушать</button>
```

Тем самым можно избавиться от проблем в будущем